src/components/ui/Tags/PlanCollaboratorInput.svelte

```diff
@@ -0,0 +1,170 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+  import type { User, UserId } from '../../../types/app';
+  import type { Plan, PlanCollaborator, PlanCollaboratorSlim, PlanSlimmer } from '../../../types/plan';
+  import type { PlanCollaboratorTag, Tag, TagsChangeEvent } from '../../../types/tags';
+  import type { ActionArray } from '../../../utilities/useActions';
+  import PlanCollaboratorInputRow from './PlanCollaboratorInputRow.svelte';
+  import TagsInput from './TagsInput.svelte';
+
+  export let collaborators: PlanCollaboratorSlim[] = [];
+  export let users: UserId[] = [];
+  export let plans: PlanSlimmer[] = [];
+  export let plan: Plan;
+  export let user: User | null;
+  export let use: ActionArray = [];
+
+  const dispatch = createEventDispatcher<{
+    create: PlanCollaborator[];
+    delete: string;
+  }>();
+
+  let inputRef: TagsInput | null;
```

Comment:
Unless you're trying to keep them separated for code flow purposes, do you mind grouping the `let` declarations and the reactive statements for consistency?

Reply:
Good catch